config/resources/domain.json

```diff
@@ -79,8 +79,9 @@
       "relationships": {
         "concepts": {
           "predicate": "skos:inScheme",
-          "target": "concept",
-          "cardinality": "many"
+          "target": "skosConcept",
```

Comment:
if this wasn't here before, we will probably need a migration to migrate the old data?

Reply:
I mean we didnt have any data that used the previous relationship

Reply:
I guess Ideally we need codelist options inherit from skosConcept and then we would need to run a migration

Reply:
but actually im not sure how inheritance and relationships work together.... gonna look at some triples I guess

Reply:
I guess if the codelist options are no different than a concept so they shouldn't change, just have to merge then in concepts and expose a single endpoint

Reply:
options

Reply:
no migrations needed I dont think